package.json

```diff
@@ -50,6 +52,6 @@
     "protractor": "~5.1.2",
     "ts-node": "~4.1.0",
     "tslint": "^5.9.1",
-    "typescript": "^2.5.3"
+    "typescript": "<3.3.0"
```

Comment:
Why `<`?

Reply:
angular-compiler/cli has some peer dependency on typescript version which requires version < 3.3.0
2.5.3 was failing for latest angular/cli version.

Reply:
I mean why `<`,  isn't latest version of CLI will support latest version of typescript as well? So it can be `>` or `~`

Reply:
yeah, CLI supports latest version but some packages has **peer dependency** over **angualr-compiler/cli version** **5.2.0** which itself not supporting typescript versions **>= 3.3.0**

Reply:
Typescript version "~3.2.0" should also work.